docs/security.md

```diff
@@ -0,0 +1,154 @@
+# Security
+
+## Overview
+
+### Role based security
+
+Role-based security allows Virtocommerce administrators to group users into roles for security purposes. Each role can be defined with specific granular permissions to allow access to different functionality within the software.
+
+Security system is composed of three main entities
+
+1. Permissions is a basic unit in security system. It describes particular right of action and has a string presentation which is used in permission checks.
+
```

Comment:
What does it mean"string presentation"? It will be great if a "string presentation" example is presented